sigle/src/modules/editor/ShortcutsDialog.tsx

```diff
@@ -96,7 +94,7 @@ export const ShortcutsDialog = () => {
             <TabsTrigger value="hints">Hints</TabsTrigger>
           </TabsList>
 
-          <TabsContent css={{ height: 650 }} value="shortcuts">
+          <TabsContent css={{ height: 600 }} value="shortcuts">
```

Comment:
fixed height here is to stop the height adjusting for each shortcuts tab creating undesirable UX - not sure if this is the best solution but works for now